src/com/duckduckgo/mobile/android/activity/OnboardingActivity.java

```diff
@@ -115,12 +109,7 @@ public void onPageScrollStateChanged(int state) {
         });
         viewPager.setAdapter(adapter);
 
-        int[] backgroundColors = new int[] {
-                ContextCompat.getColor(this, PrivacyFragment.BACKGROUND_COLOR),
-                ContextCompat.getColor(this, NoAdsFragment.BACKGROUND_COLOR),
-                ContextCompat.getColor(this, NoTrackingFragment.BACKGROUND_COLOR),
-                ContextCompat.getColor(this, RightFragment.BACKGROUND_COLOR),
-                ContextCompat.getColor(this, EndOnboardingFragment.BACKGROUND_COLOR)};
+        int[] backgroundColors = Onboarding.getBackgroundColors(this);
```

Comment:
Don't worry about converting it, it's good as a FragmentPagerAdapter. I have made another suggestion for your config which may help you solve this too. If not I'm happy to leave this one.